### PR TITLE
O/noun 68 prev next token nav widget

### PR DESCRIPTION
**Pull request by olegakbarov**
navigation between nouns

### CODE REVIEW

**Review by isaac-zora — APPROVED** [review]
Note:
Chatted with Oleg about including Date. Feedback is that there isn't a way to reliably get the date for each token, especially since mint intervals vary project to project. Projects are able to show Date because they most likely hardcoded them + the fact that they don't have to surface other projects.

File: compositions/NFTPage/NFTInfoSidebar.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export function NFTInfoSidebar({ ...props }: NFTInfoSidebar) {
   }, [activeAuctionId, tokenId])
 
   const hasPrevoiusNft = useMemo(() => {
```
Comment (bjfresh) [inline compositions/NFTPage/NFTInfoSidebar.tsx:27]: should be hasPreviousNFT ( typo on your fixed typo commit :) )

File: compositions/Header/Header.css.ts [hunk 2]
```diff
@@ -76,6 +76,16 @@ export const manageButton = style([
   }),
 ])
 
+export const modalWrapper = style({
+  overflowY: 'scroll',
```
Comment (bjfresh) [inline compositions/Header/Header.css.ts:80]: overflowY can be an atom

File: compositions/NFTPage/NFTInfoSidebar.tsx [hunk 3]
```diff
@@ -1,25 +1,50 @@
...
+      : false
+  }, [activeAuctionId, tokenId])
+
+  const hasPrevoiusNft = useMemo(() => {
+    return !!tokenId && parseInt(tokenId) >= 0
+  }, [tokenId])
+
+  const handleNext = useCallback(() => {
+    if (hasNextNft) {
+      tokenId && router.push(`/collections/${contractAddress}/${parseInt(tokenId) + 1}`)
```
Comment (bjfresh) [inline compositions/NFTPage/NFTInfoSidebar.tsx:33]: Are you verifying that there is a next tokenId in the collection? It looks like hasNextNFT is checking against the auctionId and not the collection token count.

File: compositions/NFTPage/NFTInfoSidebar.tsx (hunk 1 above)
Comment (bjfresh) [reply compositions/NFTPage/NFTInfoSidebar.tsx:27]: Couple of things here.
1) According to Matt, sometimes collection tokenId starts with 1 and sometimes with zero
2) We need to know this so the button can be disabled. It's currently possible to go to -1:

https://www.loom.com/share/489b5841b3214fcc97ffaf49d056de77

File: compositions/NFTPage/NFTPage.css.ts [hunk 4]
```diff
@@ -194,3 +194,20 @@ export const nftAttribute = style([
     },
   }),
 ])
+
+export const nftNextButton = style([
+  {
+    borderRadius: '50%',
+    padding: 0,
+    marginLeft: '.5rem',
```
Comment (bjfresh) [inline compositions/NFTPage/NFTPage.css.ts:202]: Many of these can be implemented atoms to reduce the CSS footprint.

eg 
borderRadius: round
padding: 'x0'
display
fontSize

Is the spacing based on the zord spacing? If so, marginLeft, width, and height should all be atoms too.

File: compositions/NFTPage/NFTInfoSidebar.tsx (hunk 3 above)
Comment (olegakbarov) [reply compositions/NFTPage/NFTInfoSidebar.tsx:33]: `hasNextNft` does it

**Review by bjfresh — COMMENTED** [review]
LGTM!

Couple of small nits with potential atoms but nothing to hold us back. Nice!

File: compositions/NFTPage/NFTPage.css.ts (hunk 4 above)
Comment (olegakbarov) [reply compositions/NFTPage/NFTPage.css.ts:200]: addressed, when possible
padding: 'x0' is buggy, discussed here https://ourzora.slack.com/archives/C03KSUKQX0U/p1661902550137249

**Review by bjfresh — APPROVED** [review]

**Pull request merged**